# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/HostWebGLContext.cpp
# Commit: e1b0906509ef
# Full Hash: e1b0906509efa6433978b9f980ec639b7eafbc6d
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-01-09 03:40:18
# Regressor Bug: 1477756
# File Overlap Count: 1
# Description:
#   Bug 1477756 - Fix non-webgl CI tests. r=handyman
#   
#   * Revert some partial webgl+oop+vr code.
#   * More missing FuncScope.
#   * Fix compile errors.
# ==============================================================================

diff -r 7e2a2b1b416f -r e1b0906509ef dom/canvas/HostWebGLContext.cpp
--- a/dom/canvas/HostWebGLContext.cpp	Wed Jan 08 22:19:23 2020 +0000
+++ b/dom/canvas/HostWebGLContext.cpp	Wed Jan 08 22:19:26 2020 +0000
@@ -100,6 +100,11 @@
   (void)mOwnerData.outOfProcess->mParent.SendJsWarning(text);
 }
 
+RefPtr<layers::SharedSurfaceTextureClient> HostWebGLContext::GetVRFrame()
+    const {
+  return mContext->GetVRFrame();
+}
+
 //////////////////////////////////////////////
 // Creation
 